src/en/wizden-staff/maintainer/adding-new-maintainers.md

```diff
@@ -0,0 +1,36 @@
+
+## Requirements
+
+As maintainers represent wizden as staff, certain requirements must be met before a person can be a maintainer. These requirements are as follows:
+- Be a member of the Space Station 14 Discord server, as many discussions happen there.
+- Not have had any singificant administrative issues in the past, such as being banned from the game or Discord server or having a history of being rude to other members of the community.
+    - This is a subjective requirement, and should be discussed in the voting process.
+- Have a good understanding of the codebase and how it works.
+    - This is a subjective requirement, and should be discussed in the voting process.
+
+### Voting Process
+
+Before a vote can start, the person being voted on must be nominated by a current maintainer. Once a person has been nominated, the following process should be followed:
+1. The person being nominated should be contacted and asked if they would like to be nominated. If they agree, the nomination can proceed.
+2. A vote should be created "Internal" category on Discourse, using the Discourse template.
+3. The vote should be open for 1 week, and all current maintainers should be notified of the vote. During this time, maintainers should discuss the nomination and ask any questions they may have. If there is a clear majority in favor of the nomination, the vote can be closed early. If there is a clear majority against the nomination, the vote should be closed and the nomination should be withdrawn.
+    - If there is no clear majority, the vote should remain open for the full week.
+    - If there are any concerns about the nomination, they should be discussed in the thread and addressed before the vote is closed.
+4. At the end of the week, the vote should be closed and the results should be announced. If the vote passes, the person being nominated should be contacted and asked if they would like to accept the position. If they agree, they can be added to the maintainers list.
+
+### Special Maintainer Roles
+
+Some roles are special maintainer roles and follow a different process. These roles are as follows:
+- **Head Mapper**: The head mapper is responsible for ensuring map standards and merging mapping PRs.
+- **Art lead**: The art lead is responsible for ensuring a consistent art style and approving sprite changes.
+- **Robust Toolbox Maintainer**: Maintainers of our game engine, Robust Toolbox. 
+
+#### Starting a Vote for a Special Role
+
+The process for adding a special maintainer is the same as above, but with the following changes:
+- Only a member that is already a part of the special role can nominate a new member for that role.
+    - The only exception is when the role is empty, in which case any maintainer can nominate a new member.
+- The vote will be open for 2 weeks instead of 1 week.
```

Comment:
```suggestion
The process for adding a special Maintainer is the same as for a normal Maintainer, but with the following alterations:
1. If a special role has at least one member, only a member that is already a part of the special role can nominate a new member for that role.
2. Votes to add a special Maintainer will be open for 2 weeks instead of 1 week.
```